go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/miekg/pkcs11
+module github.com/venafi-iw/pkcs11
```

Comment:
you're changing all import paths

Reply:
sorry, was doing testing, but this has been fixed in latest commit.